packages/appstore/src/constants/utils.ts

```diff
@@ -20,3 +20,32 @@ export const getHasDivider = (current_item_index: number, list_size: number, ava
             (list_size % available_grid_columns === 0 ? available_grid_columns : list_size % available_grid_columns)
     );
 };
+
+export const getWalletCurrencyIcon = (currency: string, is_dark_mode_on: boolean) => {
```

Comment:
@heorhi-deriv We are adding many of these utils functions in each PR, I would recommend we consider a data transformation layer instead of these as it's not maintainable 🙇🏻